backend_new/prisma/seed-helpers/translation-factory.ts

```diff
@@ -0,0 +1,130 @@
+import { LanguagesEnum, Prisma } from '@prisma/client';
+
+const translations = (jurisdictionName?: string) => ({
+  t: { hello: 'Hello', seeListing: 'See Listing' },
+  footer: {
+    line1: `${jurisdictionName || 'Bloom'}`,
+    line2: '',
+    thankYou: 'Thank you',
+    footer: `${jurisdictionName || 'Bloom Housing'}`,
+  },
+  header: {
+    logoUrl:
```

Comment:
I'm surprised to see a URL in translations, do you know if we ever attempt to translate this?

Reply:
This is here so that different jurisdictions can use different logos in the email. The translation can either be connected to a jurisdiction or have null jurisdiction. And the ones tied to the jurisdiction is the override to the generic one. There is no other way to change the content dynamically in the emails so that is why it is here. Nothing in here is translated, but rather a new entry is put into the database when a new language/jurisdiction combo is added. Not the greatest system at the moment